src/main/java/my/auction/user/domain/User.java

```diff
@@ -0,0 +1,42 @@
+package my.auction.user.domain;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import my.auction.global.entity.BaseEntity;
+
+@Entity
+@Getter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "auction_user")
+public class User extends BaseEntity {
+	
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	@Column(name = "user_id", unique = true, nullable = false)
+	private Long user_id;
+	
+	@Column(name = "email", unique = true, nullable = false)
+	private String email;
+	
+	@Column(name = "password", nullable = false)
+	private String password;
+	
+	@Column(name = "point", nullable = false)
+	@Builder.Default
+	private Integer point = 0;
+	
+	@Column(name = "email_verified_time")
+	@Builder.Default
+	private LocalDateTime email_verified_time = null;
```

Comment:
java에서는 카멜케이스를 사용합니다 
```suggestion
	private LocalDateTime emailVerifiedTime = null;
```